panel/src/components/Dialogs/FiberDialog.vue

```diff
@@ -29,6 +35,7 @@ export default {
 		},
 		async onSubmit(value) {
 			let dialog = null;
```

Comment:
@distantnative I've an idea about that. Actually, while the process is in progress, how about we disable the button while preventing a new action from being made at the same time?

````js
  if (this.isProcessing === true) {
    return false;
  }

  let dialog = null;
  this.isProcessing = true;
````